src/ui/public/flot-charts/jquery.flot.log.js

```diff
@@ -0,0 +1,160 @@
+/* Pretty handling of logarithmic axes.
+Copyright (c) 2007-2014 IOLA and Ole Laursen.
```

Comment:
We need to add that license reference also to the [NOTICE.txt](https://github.com/elastic/kibana/blob/master/NOTICE.txt).